docs/user/supported_devices.rst

```diff
@@ -89,6 +89,7 @@ ath79-generic
 
 * Ubiquiti
 
+  - NanoStation M2/M5 XW
```

Comment:
```suggestion
  - NanoStation M2/M5 (XW)
```
"XW" is basically a revision identifier, so let's put it in parentheses. The same is done in OpenWrt's Device Tree.

Reply:
resolved in e97f0801073a40289e605505b8d86b312d41b5bc